modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/ValueProvidersParameterDeclarationEnricher.java

```diff
@@ -247,11 +254,19 @@ private void enrichParameter(OfValues resolverClass,
         .orElse(empty());
   }
 
-  private List<String> getRequiredParametersAliases(List<ExtensionParameter> parameterDeclarations,
-                                                    Map<String, String> parameterNames) {
-    return parameterDeclarations.stream()
-        .filter(ExtensionParameter::isRequired)
-        .map(param -> parameterNames.getOrDefault(param.getName(), param.getName()))
+  private List<ParameterModel> getParametersModel(List<ExtensionParameter> parameterDeclarations,
+                                                  Map<String, String> parameterNames,
+                                                  List<ParameterDeclaration> allParameters) {
+    Map<String, Boolean> paramsInfo = parameterDeclarations.stream()
+        .collect(toMap(param -> parameterNames.getOrDefault(param.getName(), param.getName()), ExtensionParameter::isRequired));
```

Comment:
same as before. You're loosing the aliased name. Several test cases should have broken because of this. If not, we have a coverage problem you should fix as part of this PR